core/command/create.js

```diff
@@ -55,6 +66,11 @@ async function handler ({ cwd, directory: folderName = 'slides', yes: isDefault
 
   if (isDefault) {
     Object.assign(options, defaultParams)
+  } else if (params[0].choices.includes(themeName) && params[1].choices.includes(aspectRatio)) {
```

Comment:
Maybe it will be more obvious if you fall with an error, if an invalid parameter is entered?

Reply:
I've tried to prepare a new approach.